samples/OpenStore365.Samples.Serverless.Auth/Functions/GetHttpHeaders.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using OpenStore365.Serverless;
+
+namespace OpenStore365.Samples.Serverless.Auth.Functions
+{
+    public static class GetHttpHeaders
+    {
+        [FunctionName(nameof(GetHttpHeaders))]
+        public static IActionResult Execute(
+            [HttpTrigger(AuthorizationLevel.Anonymous, HTTP.GET, Route = "Http/Headers")] HttpRequest request) =>
+                new OkObjectResult(request.Headers);
```

Comment:
Wyjaśnij co to robi pls

Reply:
It returns **all** HTTP headers that got to the Azure Function.